api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -51,3 +49,23 @@ Check content of the uploaded website
     Open Thyra Page    ${API_URL}/browse/${WEBSITE_ADDRESS}/index.html
     Page Should Contain    My test website!
     Page Should Contain    Decentralization is non-negotiable
+
+
+GET /all/domains 
+    ${response}=    GET
+    ...    ${API_URL}/all/domains
+    ...    expected_status=${STATUS_OK}
+    Should Contain    ${response.text}    flappy
+
+GET /my/domains/{nickname}
+    ${response}=    GET
+    ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
+    ...    expected_status=${STATUS_OK}
+    Should Contain    ${response.text}    ${website_url}
+
+
+*** Keywords ***
+Set Suite Variables
+    ${website_url}=    Generate Random String    8    [LOWER][NUMBERS]
+    ${website_url}=    Set Variable    test${website_url}
+    Set Suite Variable    ${website_url}    ${website_url}
```

Comment:
Keywords should be in the `keywords.resource` file corresponding to the part you are testing (so in that case, `web_on_chain/keywords.resource`.